compiler/rustc_typeck/src/check/place_op.rs

```diff
@@ -73,6 +76,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
         base_expr: &hir::Expr<'_>,
         autoderef: &Autoderef<'a, 'tcx>,
         index_ty: Ty<'tcx>,
+        idx_expr: &hir::Expr<'_>,
```

Comment:
nit: looks like in other functions, the type argument follows the corresponding expression argument, and this might be more consistently named `index_expr` in this function